tests/unit/utils/paths_test.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2023-2025 The pymovements Project Authors
+# Copyright (c) 2025 The pymovements Project Authors
```

Comment:
why?

Reply:
this change is already merged by #1029

Reply:
the copyright notice was autocreated by precommit since it is actually a new file. the old file was moved to dataset/_utils